Documentation/ceph-common-issues.md

```diff
@@ -31,6 +31,7 @@ If after trying the suggestions found on this page and the problem is not resolv
 * [LVM metadata can be corrupted with OSD on LV-backed PVC](#lvm-metadata-can-be-corrupted-with-osd-on-lv-backed-pvc)
 * [OSD prepare job fails due to low aio-max-nr setting](#osd-prepare-job-fails-due-to-low-aio-max-nr-setting)
 * [Failed to create CRDs](#failed-to-create-crds)
+* [Pod Using Ceph Block Storage `is still being used` Error](#pod-using-ceph-block-storage-is-still-being-used-error)
```

Comment:
How about if you add this topic instead to the ceph-csi-troubleshooting.md? There is a link to that doc a couple lines below.

Reply:
Roger.